conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosGovernanceOperations.scala

```diff
@@ -119,8 +115,8 @@ object TezosGovernanceOperations extends LazyLogging {
     * @param ec needed to compose concurrent operations
     * @return aggregated data
     */
-  private def aggregateData(db: Database)(
-      proposalsBlocks: Map[Block, ProtocolId],
+  def aggregateData(db: Database)(
```

Comment:
`proposalsBlocks` and `proposalsBallots` are still ambiguous. They specifically leave out data for the proposal governance period. I would renamed them to `activeProposalsBlocks` and `activeProposalsBallots`.

Reply:
renamed